drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -860,6 +859,44 @@ function DragAndDropBlock(runtime, element, configuration) {
         });
     };
 
+    var doAttempt = function(evt) {
```

Comment:
When you click the button, it should immediately get disabled and optionally show some kind of a spinner while the xhr request is in progress.